src/aliceVision/numeric/numeric.hpp

```diff
@@ -568,5 +568,10 @@ void SplitRange(const T range_start, const T range_end, const int nb_split,
   }
 }
 
+/// This function initializes the global state of random number generators that our current tests
+/// depend on. This makes it possible to have exactly reproducible tests without random failures.
+/// In order to introduce variation in test execution, ALICEVISION_TEST_RANDOM_SEED environment
+/// variable can be set to an integer value.
```

Comment:
please use the doxygen convention used in the project
```suggestion
/**
 * @brief This function initializes the global state of random number generators that our current tests
 * depend on. This makes it possible to have exactly reproducible tests without random failures.
 * In order to introduce variation in test execution, ALICEVISION_TEST_RANDOM_SEED environment
 * variable can be set to an integer value.
 */
```

Reply:
Fixed, thanks!